.readthedocs.yaml

```
@@ -0,0 +1,16 @@
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+version: 2
+
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.11"
+  # Build the Ansible docsite
+  commands:
+    - python -m pip install --upgrade --no-cache-dir pip
+    - python -m pip install --exists-action=w --no-cache-dir -r ./requirements.in -c ./requirements.txt
+    - python build.py
+    - mkdir -p _readthedocs/html/
+    - cp -r output/* _readthedocs/html/
```

Make use of the documented env vars to be future-proof:
```suggestion
    - mkdir -pv "${READTHEDOCS_OUTPUT}"
    - cp -rv output "${READTHEDOCS_OUTPUT}"/html
```